### PR TITLE
Update Orders Docs with sku fields

### LINKED ISSUE #127: Confirm Includes Objects
There is inconsistency between api calls on how objects are defined in both the request and response.

For example,
- Use of Contact object and Person Object ... these are exactly the same so why have both
- In Person Object, some responses return a job_title and others don't.

In the task we'll review the request/response objects and decide how to handle the inconsistency. And, also build reusable includes files.  The includes file help us to maintain a single source of truth for how an object is defined.

Where it is practical, we'll create include files for the objects (e.g. person.md) and call that object from within a parent document by using `{% include objects/person.md %}` format.

### PR DESCRIPTION
**Pull request by nickcoleman**
Updates order docs:

- Create Order Retailer: adds `expected_sku_cost` to the response
- Add Order Tracking: adds `sku_cost` to the request
- Cancel Order: removes unnecessary fields from the request

fixes #127

### CODE REVIEW

File: _ordering/add_order_tracking.md [hunk 1]
```diff
@@ -62,14 +61,17 @@ line_item_uuid
...
 ### Expected Response Codes
 
 {% include links/response_codes.md %}
 
 
 ~~~ bash
-curl -X "POST" "https://api-sandbox.cruxconnect.com/orders/tracking/70221d81-07e9-416c-b602-360391708d17/" \
-     -H 'Authorization: Token d9741c2c241b8f9b9955130ca08dbfbd891d9c84' \
+curl -X "POST" "https://api-dev.cruxconnect.com/orders/tracking/0e63ac67-7c45-454d-b2ef-bb6b5ed387c3/" \
+     -H 'Authorization: Token 1234567890' \
```
Comment (Gidgidonihah) [inline _ordering/add_order_tracking.md]: We want sandbox credentials in the docs, not dev. I'll only note here, but be sure to change all of them in the PR.

File: _ordering/create_order_retailer.md [hunk 2]
```diff
@@ -36,22 +36,22 @@ right_code: |
...
     "retailer": {
-      "name": "Crux Retailer",
-      "uuid": "93204006-fcdc-458c-8f81-13a7337992ae",
+      "name": "projectthanos",
+      "uuid": "359204cc-c2d9-4827-b739-64c335f9fbd1",
       "user": {
-        "name": "Crux User",
-        "email": "user@mycompany.com"
+        "name": " ",
+        "email": "ncoleman@projectthanos.com"
```
Comment (Gidgidonihah) [inline _ordering/create_order_retailer.md]: Oops.

File: _ordering/add_order_tracking.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: /orders/tracking/&ltorder_uuid&gt/
+title: /orders/tracking/&lt;order_uuid&gt;/
 name: Add Order Tracking - Supplier
-position: 4.91
+position: 6.10
```
Comment (Gidgidonihah) [inline _ordering/add_order_tracking.md]: Changing from 4 to 6 is expected and not a problem? Everything renders correctly?

File: _ordering/add_order_tracking.md (hunk 1 above)
Comment (nickcoleman) [reply _ordering/add_order_tracking.md]: 👌 hadn't thought of that ... good catch

File: _ordering/create_order_retailer.md (hunk 2 above)
Comment (nickcoleman) [reply _ordering/create_order_retailer.md]: Darn ... but I wanted to be famous. 🎸

File: _ordering/add_order_tracking.md (hunk 3 above)
Comment (nickcoleman) [reply _ordering/add_order_tracking.md]: The Orders order is fine ... but definitely something to watch out for
screen shot 2018-04-20 at 5 38 46 pm

Comment (nickcoleman) [reply _ordering/add_order_tracking.md]: I think the Order was fine ... but decided to shift it a bit after thinking about it.
screen shot 2018-04-20 at 5 56 18 pm

File: _ordering/add_order_tracking.md (hunk 1 above)
Comment (nickcoleman) [reply _ordering/add_order_tracking.md]: There were some other docs that had the api-dev reference ... switched them to api-sandbox as well

File: _ordering/create_order_retailer.md (hunk 2 above)
Comment (nickcoleman) [reply _ordering/create_order_retailer.md]: I removed all the references to projectthanos and projectzuul from the docs ... changed them to retailer and supplier respectfully

**Pull request merged**